crates/rslint_parser/src/syntax/function.rs

```diff
@@ -194,7 +193,16 @@ fn parse_function(p: &mut Parser, m: Marker, kind: FunctionKind) -> CompletedMar
         })
         .ok();
 
-    parse_parameter_list(p, ParameterContext::Implementation, flags)
+    let parameter_context = if kind.is_statement() && TypeScript.is_supported(p) {
+        // It isn't known at this point if this a function overload definition (body is missing) or a regular
+        // function implementation. Let's go with the laxer of the two. Ideally, these verifications should be
+        // part of a second compiler pass.
```

Comment:
```suggestion
        // It isn't known at this point if this is a function overload definition (body is missing) or a regular
        // function implementation. Let's go with the latter of the two. Ideally, these verification should be
        // part of a second compiler pass.
```

[Plural of `verification` is `verification`](https://www.wordhippo.com/what-is/the-plural-of/verification.html) 😮